README.md

```diff
@@ -1,70 +1,92 @@
-# Getting Started with Create React App
+![swrpgc banner](https://user-images.githubusercontent.com/17935770/131261369-477cff64-39e5-462a-81bf-3f55944e512e.png)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+This application acts as a companion for the D&D game: **Star Wars: Edge of the Empire**. The companion app allows you to keep track of and store your in-game characters. You may also connect and communicate with your friends and party up so that everyone can witness the dice rolls that will unfold the story in their legendary campaign.
 
-In the project directory, you can run:
 
-### `npm start`
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+![feaures](https://user-images.githubusercontent.com/17935770/131262008-e0fb73fa-3967-41bb-852d-6fd1f71b45a7.png)
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+ - User can register an account
+ - User may create a new character that contains their information, characteristics, features
+ - All of your characters can be seen in one place
+ - Dice rolls can be be configured with actual dice animation
+ - User and friends can see a live dice log and the dice rolls
+ - Friendlists where you can see if they are online and are able to chat with
+ - Make a party with all of your friends
 
-### `npm test`
+![Installation](https://user-images.githubusercontent.com/17935770/131262041-39ba027b-0d4e-44a6-b599-8f1543ae5c6c.png)Clone the project and install dependencies
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```szh 
+git clone git@github.com:SWRPG-Capstone/swrpg-fe.git
+cd swrpg-fe
+npm install 
```

Comment:
Yarn newbie question! Since we're using yarn on our end, does npm still work for installation?

Reply:
This is actually a blind spot for me. They both work off of package.json but that is something that we should look into. I stuck with NPM since that is most people's default

Reply:
Added to the 'To Research' list. Thanks!